tools/webOS/packaging/appinfo.json.in

```diff
@@ -7,7 +7,7 @@
   "title": "@APP_NAME@",
   "icon": "icon.png",
   "largeIcon": "largeIcon.png",
-  "splashBackground": "media/splash_webOS.png",
+  "splashBackground": "splashscreen.png",
```

Comment:
shouldn't this be `media/splashscreen.png`?

Reply:
Maybe...I added a symlink but I am unsure if it works.

Reply:
Doesn't seem to work: `ares-package ERR! [Tips]: The specified path does not exist <splashscreen.png>`